Main file fixes

hayukimori opened the pull request:
# Main file small fixes

## Readline lib
`readline` lib allows user to edit characters from input. It can be very useful in case the user gets the directory name wrong.

## Change of variables
`dir` was changed because it's a python function.
It was changed to `diretorio`, the translation on `dir`, that's because the code was based on PT_BR vars.

## Classes changed
The initial letters of the classes have been changed to capital letters for better understanding in future updates.

## Small fix within 'Buscar'
`usuario` was referring to the class from outside, so it was changed to `super()` to use what was inside the class declaration.

hayukimori committed:
Main file fixes

"dir" was changed because it's a python function.
It was changed to "diretorio", the translation on "dir", that's because the code was based on PT_BR vars.

The initial letters of the classes have been changed to capital letters for better understanding in future updates.

`usuario` was referring to the class from outside, so it was changed to `super()` to use what was inside the class declaration.

hayukimori committed:
fixed `readlines` to `readline`

gabuser commented:
olá, alguns erros foram encontrados no pull request: erro de importação readline, erros no super, quando voce chama o super para herdar os atributos da superclasse voce está colocando self, mas não é necessário colocar o self, uma vez que ele está herdando, o python entende que aquilo é uma parâmetro de entrada a+. perdão, está sendo usada sim ksksk

gabuser commented:
fora isso, está funcionando bem. valeu pelo pull request

gabuser committed:
Update main.py

self foi retirado, devido não precisar, pois o python entende que todos os atributos foram herdados da superclasse
